### DISCUSSION TITLE
Add sliding UX to the menu UI in the Admin Theme

### DISCUSSION THREAD

**Discussion by fMichaleczek** [OrchardCMS/OrchardCore#7126] [Ideas]
On the AdminUI, in the left menu, when I click on a category to open, there are too many elements displayed around. I have to scroll all the time to find the sub-elements. It's counterintuitive as more modules and features are added. I propose to modify the menu to slide on the elements of level 1.

Here is an example of a jQuery menu:
[Multi-level Sliding Side Menu Demos](https://www.jqueryscript.net/demo/Multi-level-Side-Menu-Plugin-jQuery/)
[Multi-level Sliding Side Menu Plugin For jQuery](https://www.jqueryscript.net/menu/Multi-level-Side-Menu-Plugin-jQuery.html)

**Comment by Skrypt** [comment]
Our menu design is based on Azure Devops Portal. 

![image](https://user-images.githubusercontent.com/3228637/95378915-a166c580-08b2-11eb-869f-1408b8178d35.png)

We are not planning changing this soon.
You can open a PR if you think having an alternate menu could be necessary. But at this point it would require to be feature complete and bulletproof for Orchard Core 1.0.
You can easily override the current admin menu views / css in a module that you could create for other people that would prefer these types of menus. IMHO, I personally think waiting on an animation to slide elements back and forth can sometimes become less productive than scrolling up and down.
